packages/control-property-editor/test/unit/App.test.tsx

```diff
@@ -182,6 +182,42 @@ test('renders warning dialog for "FE_FROM_SCRATCH" scenario', async () => {
     fireEvent.click(okButton);
 });
 
+test('renders warning message for "ADAPTATION_PROJECT" scenario', async () => {
+    const initialState = {
+        deviceType: DeviceType.Desktop,
+        scale: 1.0,
+        selectedControl: undefined,
+        outline: [],
+        filterQuery: [
+            { name: FilterName.focusEditable, value: true },
+            { name: FilterName.focusCommonlyUsed, value: true },
+            { name: FilterName.query, value: '' },
+            { name: FilterName.changeSummaryFilterQuery, value: '' },
+            { name: FilterName.showEditableProperties, value: true }
+        ],
```

Comment:
Maybe you don't need all of these because you are testing the warning message dialog only. Just an empty array would be good
```suggestion
        filterQuery: [],
```